pipelines/tasks/hisat2.wdl

```diff
@@ -33,16 +44,23 @@ task HISAT2PE {
 }
 ## run hisat2 for rsem
 ## increase gap alignment penalty to avoid gap alignment
-## --mp 1,1 --np 1 --score-min L,0,-0.1 is default paramesters when rsem runs alignment by using bowtie2
-## no-splice-alignment otherwise rsem will report partial alignment error.
+## --mp 1,1 --np 1 --score-min L,0,-0.1 is default paramesters when rsem runs alignment by using bowtie2/Bowtie
+## --mp 1,1 and --np 1 will reduce mismatching penalty to 1 for all.
+## with no-splice-alignment no-softclip no-mixed options on, HISAT2 will onlu output concordance alignment
+## --rdg 99999999,99999999 and --rfg 99999999,99999999 will give an infinity penalty to alignment with indel.As results
```

Comment:
Small typos here; space after period, not sure if the subsequent line is a continuation of this one, but if it is it should be indented to be clear.